tests/queries/0_stateless/01801_s3_cluster.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# NOTE: this is a partial copy of the 01683_dist_INSERT_block_structure_mismatch,
+# but this test also checks the log messages
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+
+${CLICKHOUSE_CLIENT_BINARY} --send_logs_level="none" -q "SELECT *  FROM s3('https://s3.mds.yandex.net/clickhouse-test-reports/*/*/functional_stateless_tests_(ubsan)/test_results.tsv',  '$S3_ACCESS_KEY_ID', '$S3_SECRET_ACCESS', 'LineAsString', 'line String') limit 100 FORMAT Null;"
```

Comment:
Not sure it is a good idea to use our mds in tests.
It may be not always accessible for everybody who run this test.
Also, it may overload server.
I think it is ok to only use integration tests. Or if we add special option to out test script which will run docker with minio.

Reply:
I spoke with @alesapin last week and we decided that we have quotas and metrics, so we will be notified when we overload the server.